### PR TITLE
597 edit group name and description

### PR DESCRIPTION
**Pull request by tcnichol**
We can now edit the name of a group.

Additionally, the edit_group route should now handle all cases of adding users, removing users, or changing editor value for users.

### CODE REVIEW

File: backend/app/routers/groups.py [hunk 1]
```diff
@@ -104,20 +104,56 @@ async def edit_group(
...
+                await group.replace()
+                # Add user to all affected Authorization entries
+                await AuthorizationDB.find(
+                    AuthorizationDB.group_ids == ObjectId(group_id),
+                ).update(
+                    Push({AuthorizationDB.user_ids: user.email}),
+                )
         try:
-            group.update(group_dict)
+            group.name = group_dict["name"]
```
Comment (longshuicy) [inline backend/app/routers/groups.py:156]: hmm can user also update the description? I only see you overwrite the name part

**Review by longshuicy — APPROVED** [review]

File: frontend/src/components/groups/EditNameModal.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	};
+
+	const handleDialogClose = () => {
+		handleClose();
+		dispatch(resetFailedReasonInline());
+	};
+
+	const onSave = async () => {
+		setLoading(true);
+		console.log(groupId, groupName,"oldname", name,'newname')
```
Comment (longshuicy) [inline frontend/src/components/groups/EditNameModal.tsx:54]: could you remove the console.log?

File: backend/app/routers/groups.py (hunk 1 above)
Comment (tcnichol) [reply backend/app/routers/groups.py:156]: I can add an update description.

File: frontend/src/components/groups/EditNameModal.tsx (hunk 2 above)
Comment (tcnichol) [reply frontend/src/components/groups/EditNameModal.tsx]: removed

**Comment by lmarini** [comment]
pytest failing

**Review by max-zilla — APPROVED** [review]

**Pull request merged**